README.rst

```diff
@@ -30,16 +30,46 @@ Pootle.  Teams can manage their files, permissions, projects, and translate
 on-line.  Files can be downloaded for offline translation.
 
 
-Installation
+Getting started
```

Comment:
IMHO this might be confusing. Can it be renamed to something like "Trying Pootle" or something in that way?

Reply:
Also, are we no longer pointing to install instructions for users that want to got directly to install instead of test run?

Reply:
i think basically the following

- abbreviated "getting started" in README
- longer getting started here
- page/s with discussion on using dev images (running tests, building assets etc)
- page/s with discussion on running in production

its gonna take some time to get all this done tho

Reply:
Ok, as long as the README has some links to those expanded use cases at the bottom of its "Getting started" section.